shinydashboard/lantern/ui.R

```diff
@@ -21,6 +21,7 @@ ui <- dashboardPage(
       menuItem("Endpoints", tabName = "endpoints_tab", icon = icon("table")),
       menuItem("Availability", icon = icon("th"), tabName = "availability_tab"),
       menuItem("Capability", icon = icon("list-alt"), tabName = "capability_tab", badgeLabel = "new", badgeColor = "green"),
+      menuItem("Performance", icon = icon("bar-chart-o"), menuSubItem("Mean Response Time", tabName = "performance_tab")),
```

Comment:
I don't think we need a sub-menu for this. If we add another performance metric, then we can go back to sub-menus.

Reply:
Sounds good, I removed the sub menu for the performance tab